### DISCUSSION TITLE
Neue Titelseite

### DISCUSSION THREAD

**Discussion by TheMrSheldon** [TrianguIator/reisefuehrer#48] [Show and tell]
Da unsere bisherige Titelseite erstmal nur ein Platzhalter ist, habe ich mich jetzt nochmal dran gesetzt, eine neue Seite zu entwerfen.
Sie sieht der anderen ziemlich ähnlich aber hat ein anderes Design.

Anmerkungen, Verbesserungsvorschläge und Ähnliches gerne hier rein schreiben.

![titelseite](https://user-images.githubusercontent.com/10571857/161818095-eabb89bb-ecf7-4d0d-8b3e-ce75205a5af1.svg)

**Comment by TrianguIator** [comment]
Vielleicht habe ich einfach keine besonders guten Augen, aber eventuell sollten wir das orange hier heller machen, damit der Kontrast noch besser wird. Ansonsten gefällt mir das Design. Wir können ggf noch schauen, ob wir das Reiseführer-Thema o.ä. unterbringen, aber vielleicht ist weniger auch mehr.

**Comment by PatrickRossel** [comment]
Aussehen ist top, nur beim Hintergrund bin ich nicht ganz sicher, ob der zum Buch passt, aber sieht super aus.
